week03/css/project.css

```
@@ -0,0 +1,316 @@
+    font-family: '국립박물관문화재단클래식B';
+    font-size: 30px;
+    color: #0D51FE;
+}
+
+.prjSummary1 {
+    margin: 20px 0 0 -10px;
+    font-family: '국립박물관문화재단클래식M';
+    font-size: 25px;
+    color: black;
+}
+
+.prjDate1 {
+    margin: 20px 0 0 5px;
+    font-family: '국립박물관문화재단클래식M';
+    font-size: 25px;
+    color: black;
+}
+
+.more1 {
+    height: 50px;
+    width: 150px;
+    background-color: #0D51FE;
+    border: none;
+    border-radius: 10px;
+    box-shadow: 0 5px 12px 0 rgba(0, 0, 0, 0.19);
+    font-size: 20px;
+    color: white;
+    vertical-align: middle;
+    white-space: nowrap;
+    font-family: '국립박물관문화재단클래식M';
+    margin: 30px 0 0 -95px;
+}
+
+.projectCard2 {
+    width: 350px;
+    vertical-align: middle;
+    position: relative;
+    box-shadow: 0 3px 10px 0 rgba(0, 0, 0, 0.19);
+    height: 500px;
+    background: #CEE3FF;
+    font-size: 30px;
+    float: left;
+    color: black;
+    padding-left: 45px;
+    margin: -455px 0 0 400px;
+    -webkit-border-radius: 5px;
+    -moz-border-radius: 5px;
+    border-radius: 15px;
+}
+
+.projectCard2 img {
+    margin: 30px 0 0 -25px;
+}
+
+.prjTitle2 {
+    margin: 20px 0 0 95px;
+    font-family: '국립박물관문화재단클래식B';
+    font-size: 30px;
+    color: #0D51FE;
+}
+
+.prjSummary2 {
+    margin: 20px 0 0 40px;
+    font-family: '국립박물관문화재단클래식M';
+    font-size: 25px;
+    color: black;
+}
+
+.prjDate2 {
+    margin: 20px 0 0 15px;
+    font-family: '국립박물관문화재단클래식M';
+    font-size: 25px;
+    color: black;
+}
+
+.more2 {
+    height: 50px;
+    width: 150px;
+    background-color: #0D51FE;
+    border: none;
+    border-radius: 10px;
+    box-shadow: 0 5px 12px 0 rgba(0, 0, 0, 0.19);
+    font-size: 20px;
+    color: white;
+    vertical-align: middle;
+    white-space: nowrap;
+    font-family: '국립박물관문화재단클래식M';
+    margin: 30px 0 0 -95px;
+}
+
+.projectCard3 {
+    width: 350px;
+    vertical-align: middle;
+    position: relative;
+    box-shadow: 0 3px 10px 0 rgba(0, 0, 0, 0.19);
+    height: 500px;
+    background: #CEE3FF;
+    font-size: 30px;
+    float: left;
+    color: black;
+    padding-left: 45px;
+    margin: -450px 0 0 400px;
+    -webkit-border-radius: 5px;
+    -moz-border-radius: 5px;
+    border-radius: 15px;
+}
+
+.projectCard3 img {
+    margin: 30px 0 0 -25px;
+}
+
+.prjTitle3 {
+    margin: 20px 0 0 95px;
+    font-family: '국립박물관문화재단클래식B';
+    font-size: 30px;
+    color: #0D51FE;
+}
+
+.prjSummary3 {
+    margin: 20px 0 0 40px;
+    font-family: '국립박물관문화재단클래식M';
+    font-size: 25px;
+    color: black;
+}
+
+.prjDate3 {
+    margin: 20px 0 0 15px;
+    font-family: '국립박물관문화재단클래식M';
+    font-size: 25px;
+    color: black;
+}
+
+.more3 {
+    height: 50px;
+    width: 150px;
+    background-color: #0D51FE;
+    border: none;
+    border-radius: 10px;
+    box-shadow: 0 5px 12px 0 rgba(0, 0, 0, 0.19);
+    font-size: 20px;
+    color: white;
+    vertical-align: middle;
+    white-space: nowrap;
+    font-family: '국립박물관문화재단클래식M';
+    margin: 30px 0 0 -95px;
+}

```

- 고유한 id와는 다르게, class는 여러 요소가 공유할 수 있습니다.
- class 이름에서 숫자를 제거하여 여러 요소에 한 번에 적용하면, 코드 재활용성을 높일 수 있습니다.

```suggestion
body main .gamelist .projectCard {
    margin: 50px 150px 200px 0;
    padding: 40px;

    text-align: center;
    vertical-align: middle;

    font-size: 30px;
    color: black;

    background: #CEE3FF;
    box-shadow: 0 3px 10px 0 rgba(0, 0, 0, 0.19);
    -webkit-border-radius: 5px;
    -moz-border-radius: 5px;
    border-radius: 15px;
}
body main .gamelist .projectCard img {}
body main .gamelist .prjTitle {
    font-family: '국립박물관문화재단클래식B';
    font-size: 30px;
    color: #0D51FE;
}
body main .gamelist .prjSummary {
    font-family: '국립박물관문화재단클래식M';
    font-size: 25px;
    color: black;
}
body main .gamelist .prjDate {
    font-family: '국립박물관문화재단클래식M';
    font-size: 25px;
    color: black;
}
body main .gamelist button {
    width: 150px;
    height: 50px;
    background-color: #0D51FE;
    border: none;
    border-radius: 10px;
    box-shadow: 0 5px 12px 0 rgba(0, 0, 0, 0.19);
    font-family: '국립박물관문화재단클래식M';
    font-size: 20px;
    color: white;
}
```